lib/colorize.rb

```diff
@@ -77,37 +74,37 @@ def set_color_parameters( params )
   #   puts "This is blue text on red".blue.on_red.blink
   #   puts "This is uncolorized".blue.on_red.uncolorize
   #
-  def colorize( params )
+  def colorize(params)
     return self unless STDOUT.isatty
 
     begin
-      require 'Win32/Console/ANSI' if RUBY_PLATFORM =~ /win32/
+      require "Win32/Console/ANSI" if RUBY_PLATFORM.match?(/win32/)
```

Comment:
`String#match?` did not exist until in Ruby 2.4. Currently we don't control what versions of Ruby are allowed to use raygun, so this has a small chance of being a breaking change, if someone is using Ruby < 2.4.

Given that Ruby < 2.4 is official EOL, I think it is fine to require Ruby 2.4+ for raygun. We can do that by adding a line to our gem spec:

```ruby
gem.required_ruby_version = ">= 2.4.0"
```

I think that warrants a separate PR for visibility. Do you mind opening one?